charts/capsule-proxy/templates/deployment.yaml

```diff
@@ -38,7 +38,7 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" .Chart.AppVersion) }}"
```

Comment:
Maybe we could use something like this?

https://github.com/clastix/capsule/blob/a2109b74ef24ab5afe25379f3fb2ef43e916766c/charts/capsule/templates/_helpers.tpl#L64-L69

Reply:
Sounds good. I'll update the PR.